release/prologue.sh

```diff
@@ -63,7 +63,25 @@ Upstream URL:
     ${CLR_PUBLIC_DL_URL}
 Upstream Bundles:
     ${CLR_BUNDLES:-"All"}
+EOL
 
+echo
+echo '== Signing =='
+echo 'sign_update function exists:'
```

Comment:
s/exists/provided/ for the logging. Maybe even: "Update Custom Signing provide?" similar to "Is this an upstream?"